soccer/Robot.cpp

```diff
@@ -587,12 +618,51 @@ void OurRobot::replanIfNeeded(const Geometry2d::CompositeShape& global_obstacles
 	if (!_pathInvalidated) {
 		addText("Reusing path", Qt::white, "Planning");
 	} else {
-		Planning::Path *path = _planner->run(pos, angle, vel, _motionConstraints, &full_obstacles);
-		
+		double leadTime = *(_motionConstraints._replan_lead_time) * 1000000;
```

Comment:
Sure, I like that better.  For now, let's change the 1000000 to SecsToTimestamp, then after the merge, we can change them to function macros and do a global replace.  Issue #378.